ui/src/components/entity/citation/PaperSummary.tsx

```diff
@@ -199,7 +185,14 @@ export default class PaperSummary extends React.PureComponent<Props, State> {
         </div>
 
         <div className="paper-summary__library-error">
-          {this.state.errorMessage && this.state.errorMessage}
+          {!!this.state.errorMessage && this.state.errorMessage}
```

Comment:
This change fixes a common issue we ran into in S2--if `this.state.errorMessage` is the string `'0'`, this will render a `0` on the page.

Reply:
excellent, and great catch! thank you for fixing this.